packages/lesswrong/server/authenticationMiddlewares.ts

```diff
@@ -283,7 +283,7 @@ export const addAuthMiddlewares = (addConnectHandler) => {
       services: {
```

Comment:
I think I'd prefer this to factor out the `profile.emails?.[0].value` into a variable so that there wasn't a risk of mistyping it when/if people made future code changes.

Reply:
Would be nice but slightly annoying to do given the shape of the function